confd/tests/mock_data/calicoctl/explicit_peering/local-as-global-ipv6/delete.yaml

```diff
@@ -0,0 +1,37 @@
+kind: BGPPeer
+apiVersion: projectcalico.org/v3
+metadata:
+  name: bgppeer-1
+spec:
+  peerIP: 10.192.0.3
+  asNumber: 64567
+
+---
+kind: BGPPeer
+apiVersion: projectcalico.org/v3
+metadata:
+  name: bgppeer-2
+spec:
+  peerIP: 10.192.0.4
+  asNumber: 64567
+
+---
+kind: IPPool
+apiVersion: projectcalico.org/v3
+metadata:
+  name: ippool-1
+spec:
+  cidr: 192.168.0.0/16
```

Comment:
There is a mismatch between this file and the input one. However, it seems not important for the delete operation.